docs/cli/usage/tags.md

```diff
@@ -29,6 +29,9 @@ The structure of the file is the following:
   }
 ]
 ```
+## Use tags with projects created before CLI version 4.28.0
```

Comment:
Actually. Maybe it's better to just have this as a `> Note: For projects created before CLI...`